fetch.bs

```diff
@@ -5129,6 +5154,13 @@ these steps:
      <a>filtered response</a>; otherwise to <var>response</var>'s
      <a for="filtered response">internal response</a>.
 
+     <!-- the serviceworker spec is responsible for emmitting the WebDriver BiDi
+     request events in this case. That's necessary to ensure that the events are
+     only generated if the serviceworker will handle the fetch, and to get the
+     correct event ordering in the case of network fallback -->
```

Comment:
> Service Workers is responsible

and later on

> the service worker will handle

Also please align indentation with similar multiline comments (also applies elsewhere).